Doc/library/tkinter.colorchooser.rst

```diff
@@ -0,0 +1,28 @@
+:mod:`tkinter.colorchooser` --- Color choosing dialog
+=====================================================
+
+.. module:: tkinter.colorchooser
+   :platform: Tk
+   :synopsis: Color choosing dialog
+
+**Source code:** :source:`Lib/tkinter/colorchooser.py`
+
+--------------
+
+The :mod:`tkinter.colorchooser` module provides the :class:`Chooser` class
+as an interface to the native color picker dialog available in Tk 4.2 and
+newer. :class:`Chooser` implements a modal color choosing dialog window. The
```

Comment:
would suggest dropping the "available in tk 4.2 and newer"

Reply:
Yes, I believe tkinter requires 8.4.